src/main/java/com/marklogic/kafka/connect/source/QueryHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.marklogic.kafka.connect.source;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.datamovement.RowBatcher;
+import org.apache.kafka.common.config.ConfigException;
+import org.springframework.util.StringUtils;
+
+import java.util.Map;
+
+import static java.lang.String.format;
+
+public interface QueryHandler {
+    void addQueryToRowBatcher(RowBatcher<?> rowBatcher);
+
+    String updatePreviousMaxConstraintColumnValue(long queryStartTimeInMillis);
+
+    String ANY_QUOTES_REGEX_PATTERN = "[\"']";
+    default String sanitize(String target) {
```

Comment:
The sanitize and removeWhitespace methods can live here for now, but because they're not part of the public interface, they should be elsewhere, like in a non-public QueryHandlerUtil class. The newQueryHandler method is public though as it's the expected factory method for obtaining a QueryHandler.